src/main/twirl/com/lightbend/lagom/docs/getstartedjava.scala.html

```diff
@@ -52,48 +79,66 @@
                     </ul>
 
                     <p>
-                        See <a href="@context.path/documentation/@context.currentDocsVersion/java/JavaPrereqs.html">detailed instructions</a>
-                        for help setting up the prerequisites and getting started. Or, if you are raring to go, just follow these simple steps:
+                        If you have the prerequisites installed and are familiar with Maven, 
+                        create a Hello World project using the commands below. Otherwise, see <a href="@context.path/documentation/@context.currentDocsVersion/java/JavaPrereqs.html">detailed instructions</a>
+                        for help setting up the prerequisites and getting started.
                     </p>
 
                     <ol>
                         <li>
-                            <p>
-                                Create a new project from the command line (all on one line):
-                            </p>
+                          
+                                From a command prompt with administrative privileges, create a new project by entering the following (all on one line):
```

Comment:
"Administrative privileges" only really makes sense for Windows users. Also, I can't think of a good reason why it should be required. Have you confirmed that it _doesn't_ work if you _don't_ have administrative privileges?

If that's the case, I think we should fix the problem.